### PR TITLE
Thanos multicluster metrics mvp docs

### PR DESCRIPTION
**Pull request by fekete-robert**
.

### CODE REVIEW

File: docs/quickstart/multicluster.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        kind: Certificate
+        metadata:
+          name: peer-tls
+        spec:
+          secretName: peer-tls
+          commonName: peer-endpoint.cluster.notld
+          dnsNames:
+          - $PEER_ENDPOINT
+          issuerRef:
+        name: selfsigned
```
Comment (tarokkk) [inline docs/quickstart/multicluster.md]:
```suggestion
            name: selfsigned
```

**Review by tarokkk — CHANGES_REQUESTED** [review]

File: docs/quickstart/multicluster.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+          usages:
+          - server auth
+          - client auth
+        EOF
+        ```
+
+    1. Save the secret so you can load it into the peer later (for example, into the file called peer-tls.yaml).
+
+        ```bash
+        kubectl neat get secret peer-tls -o yaml > peer-tls.yaml
```
Comment (tarokkk) [inline docs/quickstart/multicluster.md]:
```suggestion
        kubectl get secret peer-tls -o yaml > peer-tls.yaml
```

File: docs/headless/warning-prometheus-label.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+---
+{{< warning >}}If Prometheus is already installed on the peer cluster, make sure that properly sets the **cluster label** for the collected metrics: the label must be present, and unique, otherwise the metrics of the different clusters become mixed up.
+If One Eye has been installed on the peer cluster, make sure that the name of the observer custom resource is different on the observer and the peer clusters.
```
Comment (pepov) [inline docs/headless/warning-prometheus-label.md]:
```suggestion
If One Eye has been installed on the peer cluster, make sure that the `spec.clusterName` field of the observer custom resource is different on the observer and the peer clusters. Latest version of `One Eye` (>=0.5.0) tries to detect it automatically from the context.
```

File: docs/quickstart/multicluster.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
+---
+title: Multicluster setup
+weight: 200
+---
+
+The following procedure shows you how to collect metrics from a peer cluster to an existing One Eye deployment. The cluster running the One Eye deployment is called the **observer cluster**, the new cluster you are collecting metrics from is called the **peer cluster**. For details on how One Eye collects metrics from multiple clusters, see {{% xref "/docs/one-eye/multicluster/_index.md" %}}.
+
+## Prerequisites
+
+- You have completed the steps described in {{% xref "/docs/one-eye/quickstart/_index.md" %}} and you have a working One Eye deployment.
```
Comment (pepov) [inline docs/quickstart/multicluster.md]: we don't need the full quickstart here as the steps describe the full setup process.

File: docs/quickstart/multicluster.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+        ```bash
+        cat <<EOF | kubectl apply -f-
+        apiVersion: v1
+        kind: Secret
+        metadata:
+          name: peer-tls
+          labels:
+            monitoring.banzaicloud.io/thanospeer: ${PEER_ENDPOINT}
+            monitoring.banzaicloud.io/thanospeer-ca: ${PEER_ENDPOINT}
```
Comment (pepov) [inline docs/quickstart/multicluster.md:76]: we should highlight the fact that these labels are important for the thanospeer to be able to use them automatically

File: docs/whatsnew/_index.md [hunk 6]
```diff
@@ -4,6 +4,10 @@ shorttitle: What's new
 weight: 15
 ---
 
+## Release 0.5 (2021-04-19) {#oe05}
+
+One Eye allows you to attach multiple peer clusters to a central observer cluster to make your metrics available centrally, and store in object storage in a cost and storage-efficient way. One Eye automates several steps of connecting and maintaining the peer clusters, making it possible to operate your monitoring infrastructure with a low overhead. For details on how One Eye collects metrics from multiple clusters, see {{% xref "/docs/one-eye/multicluster/_index.md" %}}.
+
```
Comment (pepov) [inline docs/whatsnew/_index.md:10]: we should also add external ingress + authentication with pomerium + role based authorization as a note here, although we don't have the docs for it just yet

**Review by tarokkk — APPROVED** [review]
Nice job thanks

**Pull request merged**